### ISSUE TITLE
Body scroll while dragging?

### ISSUE THREAD

**Issue by ispal** [kutlugsahin/vue-smooth-dnd#18]
Hi,

Just wanted to ask how the scroll works? I have checked the examples and if I remove `overflow: auto` from the demo (.demo), it stops working. I wonder if it's possible to make the whole page (body) to scroll while dragging?

**Comment by kutlugsahin** [comment]
@ispal 
Yes, library does not support body scrolling for now. I will add this feature in upcoming versions. For now just try to put it inside a scrollable container.

**Comment by aariacarterweir** [comment]
Hi @kutlugsahin, love this! However can you confirm that body (window) scrolling is still working? Even just testing your demo page it doesn't appear to function. :)

**Comment by DanielQuero** [comment]
Hi,
I have been looking for this feature in the latest versions and it is broken after 0.2.8. The weird thing is that the demo page linked on readme is working fine, so I don't know if the demo page is an older version or I need to do something more.

**Comment by ffxsam** [comment]
Body scrolling still doesn't work in version 0.8.1.

**Comment by kutlugsahin** [comment]
To workaround this use a scrollable div element instead of instead of body.

**Comment by you-yas55** [comment]
Just adding to the above, it also seems that the scroll behaviour does not work when using `overflow: overlay;` any way to get around that?

**Comment by mkegoldberg** [comment]
If it's not working, you might need to set a height value on the scrollable container.

**Comment by isaac-souza** [comment]
> To workaround this use a scrollable div element instead of instead of body.

```vue
<template>
    <div class="overflow-y-auto h-screen">
        <span>Studio Ghibli Tier List</span>
        <Container orientation="vertical" @drop="onDrop" :auto-scroll-enabled="true">
            <Draggable v-for="(item, i) in items" :key="item.id">
                <div class="w-24 h-24 bg-blue-400 m-2">
                    {{ i + 1 }} -> {{ item.data }}
                </div>
            </Draggable>
        </Container>
    </div>
</template>
```

Not sure if anyone is still having that issue, but as answered by @[kutlugsahin](https://github.com/kutlugsahin), the workaround works just fine, at least it worked for me.

It is on the second line ```<div class="overflow-y-auto h-screen">```.

Hope that helps anyone.
